src/Components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './NavBar.css';
+
+/**
+ * NavBar component for the app
+ */
+export default function NavBar() {
+  return (
+    <nav className="NavBar">
+      <NavLink exact to="/" activeClassName="nav-selected">
```

Comment:
For the list view item, maybe we can change the URL to `/list-view` vs `/` when you navigate back to list view? @edenbekele and I feel that it would be best to keep the `/` as the home page of the application and keep list view separate.

Reply:
We have completed updating `/` to `/list ` to keep the homepage free for future use.